mava/adders/base.py

```diff
@@ -51,14 +51,14 @@ def add(
         self,
         actions: Dict[str, types.NestedArray],
         next_timestep: dm_env.TimeStep,
-        extras: Dict[str, types.NestedArray] = {"": ()},
+        next_extras: Dict[str, types.NestedArray] = {"": ()},
```

Comment:
Do we need `{"": ()}`? Can't we just have an empty dict?

Reply:
Sure, I will change this.